packages/web/src/utils/zIndex.ts

```diff
@@ -31,14 +31,15 @@ export enum zIndex {
   // Web3 wallet connect modal
   MUSIC_CONFETTI = 10000,
   WEB3_WALLET_CONNECT_MODAL = 10001,
+  MODAL_OVERFLOW_MENU_POPUP = 10001,
 
   ARTIST_POPOVER_POPUP = 20000,
 
   PLAY_BAR_POPUP_MENU = 20001,
 
   FEATURE_FLAG_OVERRIDE_MODAL = 30000,
 
-  MODAL_OVERFLOW_MENU_POPUP = 10001
+  SOMETHING_WRONG_PAGE = 40000
```

Comment:
Doing a quick search through the workspace for `z-index:` shows a few items with values greater than 40000, most notably [the stripe onramp](https://github.com/AudiusProject/audius-protocol/blob/be5c877e64bade5c6c515e9d3e28c5ca36def1d4/packages/web/src/components/stripe-on-ramp-modal/StripeOnRampModal.module.css#L2) (😄 )

Since this is meant to show above all other things, we may need to adjust those values downward or make this an even ridiculously higher number to ensure it always shows above the app content?